src/main.js

```diff
@@ -28,19 +20,22 @@ function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
 
-function displayNewCover() {  
-  bookCover.src = covers[getRandomIndex(covers)];
-  bookTitle.innerText = titles[getRandomIndex(titles)];
-  firstDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
-  secondDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
+var currentCover;
+function displayNewCover(placeHolder) {
+  bookCover.src = placeHolder.cover || covers[getRandomIndex(covers)];
```

Comment:
Rename placeHolder to something more appropriate like currentCover or workingCover.